tests/archive.rs

```diff
@@ -51,3 +51,44 @@ fn archives_are_disabled(tmpdir: TempDir, port: u16) -> Result<(), Error> {
 
     Ok(())
 }
+
+#[rstest]
+fn test_tar_archives(tmpdir: TempDir, port: u16) -> Result<(), Error> {
+    let mut child = Command::cargo_bin("miniserve")?
+        .arg(tmpdir.path())
+        .arg("-p")
+        .arg(port.to_string())
+        .arg("-g")
+        .stdout(Stdio::null())
+        .spawn()?;
+
+    sleep(Duration::from_secs(1));
+
+    // Ensure the links to the tar archive exists and tar not exists
```

Comment:
One small thing I just found:
`// Ensure the links to the compressed tar archive exists and link to plain tar doesn't exist`

Reply:
I am happy to fix this. But since you already merge this, should I still change this?